Model/AHK.cs

```diff
@@ -10,92 +10,159 @@
                             Interop.PostMessage(roClient.process.MainWindowHandle, Constants.WM_LBUTTONUP, 0, 0);
-                            Thread.Sleep(this.ahkDelay);
+                            Thread.Sleep(this.AhkDelay);
                         }
                     }
                     else
                     {
-                        while (Keyboard.IsKeyDown(config.key))
+                        while (Keyboard.IsKeyDown(config.Key))
                         {
                             Interop.PostMessage(roClient.process.MainWindowHandle, Constants.WM_KEYDOWN_MSG_ID, thisk, 0);
-                            Thread.Sleep(this.ahkDelay);
+                            Thread.Sleep(this.AhkDelay);
                         }
                     }
 
-                    Thread.Sleep(this.ahkDelay);
+                    Thread.Sleep(this.AhkDelay);
+                }
+            }
+            return 0;
+        }
+
+        private int AHKCompatibilityWithoutFlick(Client roClient)
+        {
+            foreach (KeyConfig config in AhkEntries.Values)
+            {
+                Keys thisk = (Keys)Enum.Parse(typeof(Keys), config.Key.ToString());
+                if (!Keyboard.IsKeyDown(Key.LeftAlt) && !Keyboard.IsKeyDown(Key.RightAlt))
+                {
+                    if (config.ClickActive && Keyboard.IsKeyDown(config.Key))
+                    {
+                        while (Keyboard.IsKeyDown(config.Key))
+                        {
+                            Interop.PostMessage(roClient.process.MainWindowHandle, Constants.WM_KEYDOWN_MSG_ID, thisk, 0);
+                            Interop.PostMessage(roClient.process.MainWindowHandle, Constants.WM_LBUTTONDOWN, 0, 0);
+                            Thread.Sleep(1);
+                            Interop.PostMessage(roClient.process.MainWindowHandle, Constants.WM_LBUTTONUP, 0, 0);
+                            Thread.Sleep(this.AhkDelay);
+                        }
+                    }
+                    else
+                    {
+                        while (Keyboard.IsKeyDown(config.Key))
+                        {
+                            Interop.PostMessage(roClient.process.MainWindowHandle, Constants.WM_KEYDOWN_MSG_ID, thisk, 0);
+                            Thread.Sleep(this.AhkDelay);
+                        }
+                    }
+
+                    Thread.Sleep(this.AhkDelay);
+                }
+            }
+            return 0;
+        }
+
+        private int AHKSpeedBoost(Client roClient)
+        {
+            foreach (KeyConfig config in AhkEntries.Values)
+            {
+                Keys thisk = (Keys)Enum.Parse(typeof(Keys), config.Key.ToString());
+                while (Keyboard.IsKeyDown(config.Key))
+                {
+                    Interop.PostMessage(roClient.process.MainWindowHandle, Constants.WM_KEYDOWN_MSG_ID, thisk, 0);
```

Comment:
We can put this code inside a method and reuse in `AHKSpeedBoost` and `AHKCompatibility` right?